### ISSUE TITLE
Three is no image in the rviz.

### ISSUE THREAD

**Issue by yileichen123** [LidarPerception/kitti_ros#1]
I had download the kitti data and I had all the requirements.But there is no images and points on the screen. And there is no error.It just show and pause as the following:
[ 2011-09-26 13:04:32.335337 ] # of Point Clouds: 493588
###########
[INFO] Show image:  0000000000.png

And the url: ROS_MASTER_URI=http://localhost:11311, I can't cotact on the website

**Comment by maxwellfrom** [comment]
I met the same problem and the ros showed that "[kitti_player-2] process has died [pid 24626, exit code 1", I think I have satisfied all the requirements .

**Comment by Durant35** [comment]
Have you press the keyboard to control the data replay？

**Comment by maxwellfrom** [comment]
Yes, I have tried to use keyboard, but it didn't work. @Durant35

**Comment by maxwellfrom** [comment]
just stoped at [ 2011-09-26 13:04:32.335337 ] # of Point Clouds: 493588
###########
[INFO] Show image:  0000000000.png

**Comment by Durant35** [comment]
Could you please upload a screenshot?

**Comment by maxwellfrom** [comment]
![screenshot from 2019-01-11 20-43-11](https://user-images.githubusercontent.com/30398190/51034544-0f992e00-15e2-11e9-9414-0d4d42edf509.png)

**Comment by maxwellfrom** [comment]
The same situation appeared when I used 'quickstart '

**Comment by maxwellfrom** [comment]
This is the rviz screenshot :
![screenshot from 2019-01-11 20-54-08](https://user-images.githubusercontent.com/30398190/51034887-2f7d2180-15e3-11e9-9385-16fc91317417.png)
